RELEASE_NOTES.md

```diff
@@ -6,6 +6,16 @@
 
 ChefDK now ships with Chef 14.7.17. See <https://docs.chef.io/release_notes.html> for more information on what's new.
 
+## Updated Tooling
+
+### Berkshelf
+
+Berkshelf has been updated to 7.0.7 and includes the following enhancements:
+
+* Relative URL now supported in `location_path`
+* `berks outdated` now supports an `--all`  flag which will include outdated dependencies that won't
+  satisfy version constraints set in the Berksfile.
```

Comment:
Line 16 and 17 should be merged together.
Also, there should be a comma between "flag which" in line 16.

Reply:
:+1: thanks